applications/serial_lte_modem/src/nrfcloud/slm_at_nrfcloud.c

```diff
@@ -703,8 +705,7 @@ static int handle_at_nrf_cloud_pos(enum at_cmd_type cmd_type,
 
 			/* Parse the MAC address. */
 			len = sizeof(mac_addr_str);
-			err = at_params_string_get(
-				param_list, param_idx, mac_addr_str, &len);
+			err = at_parser_string_get(parser, param_idx, mac_addr_str, &len);
```

Comment:
This seems to be broken. Please run the TestSLMnRFCloud tests for SLM.

Compile with:

python3 /home/<username>/ncs/test-fw-nrfconnect-nrf_iot/utils/oulu/ci/slm/build_images.py /home/<username>/ncs/nrf/applications/serial_lte_modem/ nrf9160dk/nrf9160/ns -i nrf_cloud